python/katana/local/_graph.pyx

```diff
@@ -388,3 +369,23 @@ cdef class Graph(GraphBase):
         Internal.
         """
         return <uint64_t>self.underlying_property_graph()
+
+
+def from_graphml(path, uint64_t chunk_size=25000):
```

Comment:
Because this is Cython and the other file is Python. I cannot just copy it there. I could create an import data Cython file for it if you think that's a good idea.

Reply:
I see, I need to understand cython but yeah if there is a way to put this method outside this file I think it's good idea since it doesn't really belong in this file intuitively right? If it's too much work than I think it's okay.